registry/domain/models/service_group.py

```diff
@@ -25,9 +25,9 @@ def to_dict(self):
             "daemon_addresses": self._daemon_address
         }
 
-    def _get_endpoints(self, endpoint_dict):
+    def _get_endpoints(self):
         endpoints = []
```

Comment:
you have changed the contract, did you check the places it is being used

Reply:
yeah.. the only place it is being used it `ServiceGroup.to_metadata()`